samples/sampler/stories/qa/VirtualGridList.js

```diff
@@ -108,7 +108,7 @@ class MyVirtualList extends Component {
 	};
 
 	render () {
-		let props = {...this.props};
+		let {scrollMode, ...props} = this.props;
```

Comment:
It's ok but we usually name the rest props to `rest`.

```suggestion
		const {scrollMode, ...rest} = this.props;
```

Reply:
OK